test/javascripts/acceptance/delete-channel-test.js

```diff
@@ -0,0 +1,113 @@
+import { click, fillIn, visit } from "@ember/test-helpers";
+import {
+  allChannels,
+  chatChannels,
+  chatView,
+} from "discourse/plugins/discourse-chat/chat-fixtures";
+import {
+  acceptance,
+  exists,
+  query,
+} from "discourse/tests/helpers/qunit-helpers";
+import { test } from "qunit";
+
+const baseChatPretenders = (server, helper) => {
+  server.get("/chat/:chatChannelId/messages.json", () =>
+    helper.response(chatView)
+  );
+  server.post("/chat/:chatChannelId.json", () => {
+    return helper.response({ success: "OK" });
+  });
+  server.get("/chat/lookup/:message_id.json", () => helper.response(chatView));
+  server.post("/uploads/lookup-urls", () => {
+    return helper.response([]);
+  });
```

Comment:
Minor nit here but we're mixing cases here. `:chatChannelId` and `:message_id`. I think we usually just follow snake case in URLs.